Engines/Wine/Verbs/dotnet40/script.js

```diff
@@ -3,12 +3,21 @@ include(["engines", "wine", "plugins", "override_dll"]);
 include(["utils", "functions", "net", "resource"]);
 include(["engines", "wine", "verbs", "luna"]);
 include(["utils", "functions", "filesystem", "files"]);
+include(["engines", "wine", "plugins", "windows_version"]);
 
 /**
 * Verb to install .NET 4.0
 * @returns {Wine} Wine object
 */
 Wine.prototype.dotnet40 = function () {
+    if (this.architecture = "amd64") {
+        print(tr("This package ({0}) may not fully work on a 64-bit installation. 32-bit prefixes may work better.", "dotnet40"));
+    }
+
+    var OSVersion = this.windowsVersion();
+    if (OSVersion == null)
```

Comment:
Can this actually happen?

Reply:
Yes, when you execute the script in a preInstall without this condition, the os version set at the end of the script does not work because the version is null.

Reply:
You want to use a windows windows only during the installation, correct? 
If so, you can ommit this check since a recent commit I have done. In fact, now, windowsVersion(null) will unset the setting